Forms/frmSettings.Designer.cs

```diff
@@ -63,115 +61,300 @@ private void InitializeComponent()
             this.btnClose.Name = "btnClose";
             this.btnClose.Size = new System.Drawing.Size(75, 23);
             this.btnClose.TabIndex = 0;
-            this.btnClose.Text = Properties.Resources.Settings_Close;
+            this.btnClose.Text = global::CloudFlareDDNS.Properties.Resources.Settings_Close;
             this.btnClose.UseVisualStyleBackColor = true;
             this.btnClose.Click += new System.EventHandler(this.btnClose_Click);
             // 
             // btnApply
             // 
-            this.btnApply.Location = new System.Drawing.Point(12, 156);
+            this.btnApply.Location = new System.Drawing.Point(12, 521);
             this.btnApply.Name = "btnApply";
             this.btnApply.Size = new System.Drawing.Size(75, 23);
             this.btnApply.TabIndex = 1;
-            this.btnApply.Text = Properties.Resources.Settings_Apply;
+            this.btnApply.Text = global::CloudFlareDDNS.Properties.Resources.Settings_Apply;
             this.btnApply.UseVisualStyleBackColor = true;
             this.btnApply.Click += new System.EventHandler(this.btnApply_Click);
             // 
-            // lblDomain
-            // 
-            this.lblDomain.AutoSize = true;
-            this.lblDomain.Location = new System.Drawing.Point(12, 16);
-            this.lblDomain.Name = "lblDomain";
-            this.lblDomain.Size = new System.Drawing.Size(77, 13);
-            this.lblDomain.TabIndex = 2;
-            this.lblDomain.Text = Properties.Resources.Settings_Domain;
-            // 
-            // txtDomainName
-            // 
-            this.txtDomainName.Location = new System.Drawing.Point(95, 12);
-            this.txtDomainName.Name = "txtDomainName";
-            this.txtDomainName.Size = new System.Drawing.Size(265, 20);
-            this.txtDomainName.TabIndex = 3;
-            // 
             // txtEmailAddress
             // 
-            this.txtEmailAddress.Location = new System.Drawing.Point(95, 38);
+            this.txtEmailAddress.Location = new System.Drawing.Point(95, 12);
             this.txtEmailAddress.Name = "txtEmailAddress";
-            this.txtEmailAddress.Size = new System.Drawing.Size(265, 20);
+            this.txtEmailAddress.Size = new System.Drawing.Size(350, 20);
             this.txtEmailAddress.TabIndex = 5;
             // 
             // lblEmailAddress
             // 
             this.lblEmailAddress.AutoSize = true;
-            this.lblEmailAddress.Location = new System.Drawing.Point(12, 41);
+            this.lblEmailAddress.Location = new System.Drawing.Point(12, 15);
             this.lblEmailAddress.Name = "lblEmailAddress";
             this.lblEmailAddress.Size = new System.Drawing.Size(76, 13);
             this.lblEmailAddress.TabIndex = 4;
-            this.lblEmailAddress.Text = Properties.Resources.Settings_EmailAddress;
+            this.lblEmailAddress.Text = "Email Address:";

```

Comment:
Please use globals (as above) or properties (as left) for these strings.
If anyone wants to translate the application, all strings need to be in one place.